src/app/home/home.component.scss

```diff
@@ -49,3 +49,44 @@ swiper-slide {
     padding: 0;
   }
 }
+
+.charity-carousel-navigation {
```

Comment:
code below copied and adapted from https://github.com/thebiggive/components/blob/a1f5f9542fbec587c61e18aec8c3a0c230329954/src/components/biggive-carousel/biggive-carousel.scss#L21